asapdiscovery-alchemy/asapdiscovery/alchemy/schema/fec/protocols.py

```diff
@@ -12,42 +12,16 @@
     IntegratorSettings,
     OpenMMEngineSettings,
     SimulationSettings,
-    SolvationSettings,
+    OpenMMSolvationSettings,
     SystemSettings,
 )
 from openff.units import unit as OFFUnit
+from openff.models.types import FloatQuantity
 from pydantic import Field
 
 
-class SolventSettings(_SchemaBase):
-    """
-    A settings class to encode the solvent used in the OpenFE FEC calculations.
-    """
-
-    type: Literal["SolventSettings"] = "SolventSettings"
-
-    smiles: str = Field("O", description="The smiles pattern of the solvent.")
-    positive_ion: str = Field(
-        "Na+",
-        description="The positive monoatomic ion which should be used to neutralize the system and to adjust the ionic concentration.",
-    )
-    negative_ion: str = Field(
-        "Cl-",
-        description="The negative monoatomic ion which should be used to neutralize the system and to adjust the ionic concentration.",
-    )
-    neutralize: bool = Field(
-        True,
-        description="If the net charge of the chemical system should be neutralized by the ions defined by `positive_ion` and `negative_ion`.",
-    )
-    ion_concentration: FloatQuantity["molar"] = Field(  # noqa: F821
-        0.15 * OFFUnit.molar,
-        description="The ionic concentration required in molar units.",
-    )
-
-    def to_solvent_component(self) -> gufe.SolventComponent:
-        return gufe.SolventComponent(**self.dict(exclude={"type"}))
-
-
+# DD: I think we'll want to try and avoid duplicating the attribute structure of Protocol settings here;
+# though we don't expect them to change often, it would be cleaner to avoid duplication
```

Comment:
Great point, I would love to use the openfe/gufe models here if possible and did originally try this to track the provenance of all settings used in the calculation but hit issues with serialisation see https://github.com/OpenFreeEnergy/openfe/issues/518, or found that some settings like the `SolventSettings` class above didn't have a model in openfe.